src/contact/contact.tsx

```diff
@@ -98,16 +109,60 @@ const Contact = () => {
             />
           </div>
           <div className="contactLabel">
-            <label htmlFor="socialMedia">Social Media Links</label>
+            <label htmlFor="instagram">Instagram</label>
+          </div>
+          <div>
+            <input
+              className="contactInput"
+              id="instagram"
+              name="instagram"
+              type="instagram"
```

Comment:
https://www.w3schools.com/html/html_form_input_types.asp
I think that should be type="url"